metrics/core.go

```diff
@@ -117,3 +94,56 @@ func CreateMetrics() *Metrics {
 
 	return m
 }
+
+func CreatePrometheusMetrics(cfg config.PrometheusMetrics) *PrometheusMetrics {
+	cacheWriteTimeBuckts := []float64{0.001, 0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 1}
+	registry := prometheus.NewRegistry()
+	promMetrics := &PrometheusMetrics{
+		Registry: registry,
+		RequestDurationMetrics: newHistogramVector(cfg, registry,
+			"Request duration",
+			"Duration in seconds to write to Prebid Cache labeled by get or put method and current URL or backend request type.",
+			[]string{"method", "result"},
+			cacheWriteTimeBuckts,
+		),
+		MethodToEndpointMetrics: newCounterVecWithLabels(cfg, registry,
+			"Puts and gets and GetsBackend request counts",
+			"How many get requests, put requests, and get backend requests cathegorized by total requests, bad requests, and error requests.",
+			[]string{"method", "count_type"},
+		),
+		RequestSyzeBytes: newHistogramVector(cfg, registry,
+			"Request size in bytes",
+			"Currently implemented only for backend put requests.",
+			[]string{"method"},
+			cacheWriteTimeBuckts,
+		),
+		ConnectionErrorMetrics: newCounterVecWithLabels(cfg, registry,
+			"Connection success and error counts",
+			"How many accept_errors, or close_errors connections",
+			[]string{"connections"},
+		),
+		ActiveConnections: newGaugeMetric(cfg, registry,
+			"connections.active_incoming",
+			"How many connections are currenctly opened",
+		),
+		ExtraTTLSeconds: prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Why do we track this metric?

Reply:
InfluxDB metrics were already tracking this field. Do you want us to reconsider the use of this metric in a future PR?